src/main/java/org/wso2/carbon/module/swiftiso20022/JsonToMT940Transformer.java

```diff
@@ -0,0 +1,208 @@
+import org.wso2.carbon.module.swiftiso20022.utils.JsonToMt940Utils;
+import org.wso2.carbon.module.swiftiso20022.utils.ValidatorUtils;
+
+import java.util.List;
+import java.util.Optional;
+
+import javax.xml.stream.XMLStreamException;
+
+/**
+ * Class to convert the JSON payload to MT940 format.
+ */
+public class JsonToMT940Transformer extends AbstractConnector {
+
+    // TODO:
+    public void connect(MessageContext messageContext) throws ConnectException {
+        try {
+            this.log.debug("Executing JsonToMT940Convertor to convert the JSON payload to MT940 format");
+
+            org.apache.axis2.context.MessageContext axis2MessageContext = ((Axis2MessageContext) messageContext)
+                    .getAxis2MessageContext();
+            Optional<String> payload = ConnectorUtils.buildMessagePayloadFromMessageContext(axis2MessageContext);
+
+            if (payload.isPresent()) {
+                RequestPayloadModel requestPayload = (new Gson()).fromJson(payload.get(), RequestPayloadModel.class);
+
+                ErrorModel validationResponse = validateRequestPayload(requestPayload);
+                if (validationResponse.isError()) {
+                    this.log.error(validationResponse.getErrorMessage());
+                    ConnectorUtils.appendErrorToMessageContext(messageContext, validationResponse.getErrorCode(),
+                            validationResponse.getErrorMessage());
+                    this.handleException(ConnectorConstants.ERROR_VALIDATION_FAILED, messageContext);
+                }
+
+                JSONObject mt940FormatJson = JsonToMt940Utils.appendConstructedFields(payload.get(), requestPayload);
+                ConnectorUtils.appendJsonResponseToMessageContext(messageContext, mt940FormatJson.toString());
+            } else {
+                this.log.error(ConnectorConstants.ERROR_MISSING_PAYLOAD);
+                ConnectorUtils.appendErrorToMessageContext(messageContext, ConnectorConstants.MISSING_REQUEST_PAYLOAD,
+                        ConnectorConstants.ERROR_MISSING_PAYLOAD);
+                this.handleException(ConnectorConstants.ERROR_MISSING_PAYLOAD, messageContext);
+            }
+
+        } catch (JSONException | AxisFault | XMLStreamException | ConnectException e) {
+            this.log.error(ConnectorConstants.PROCESSING_ERROR, e);
+            ConnectorUtils.appendErrorToMessageContext(messageContext, ConnectorConstants.SERVER_ERROR,
+                    ConnectorConstants.PROCESSING_ERROR);
+            throw new ConnectException(e, ConnectorConstants.PROCESSING_ERROR);
+        }
+    }
+
+    /**
+     * Method to validate the request payload.
+     *
+     * @param requestPayload  Request payload
+     * @return               Error model if there is an error, else empty error model
+     */
+    private ErrorModel validateRequestPayload(RequestPayloadModel requestPayload) {
+
+        if (requestPayload.getBlock1() == null || requestPayload.getBlock1().isBlank()) {
```

Comment:
Let's use StringUtils.isBlank() here.

Reply:
Fixed in https://github.com/wso2-extensions/mediation-swift-iso20022-module/pull/2/commits/18ec29ed3926964e6b999c6fd5d385b24c1f8fa5